InAppUtils/InAppUtils.m

```diff
@@ -146,6 +146,17 @@ - (void)paymentQueueRestoreCompletedTransactionsFinished:(SKPaymentQueue *)queue
     }
 }
 
+RCT_EXPORT_METHOD(receiptData:(RCTResponseSenderBlock)callback)
+{
+    NSURL *receiptUrl = [[NSBundle mainBundle] appStoreReceiptURL];
+    NSData *receiptData = [NSData dataWithContentsOfURL:receiptUrl];
+    if (!receiptData) {
+      callback(@[@"not_available"]);
+    } else {
+      callback(@[[NSNull null], [receiptData base64EncodedStringWithOptions:0]]);
```

Comment:
This is what is expected, yes. [The docs](https://developer.apple.com/library/ios/releasenotes/General/ValidateAppStoreReceipt/Chapters/ValidateRemotely.html#//apple_ref/doc/uid/TP40010573-CH104-SW3) explain that you need send the base 64 encoded receipt to the server.